deployment/cloud_run/test_direct_errorhandler.py

```
@@ -0,0 +1,84 @@
+            app = Flask(__name__)
+            api = Api(app, version="1.0.0", title="Test")
+            print("✅ API object created")
+        except Exception as e:
+            print(f"❌ API creation failed: {e}")
+            raise SystemExit(1)
+
+        # Let's try to register error handlers directly
+        try:
+            print("1. Testing direct error handler registration...")
+
+            def rate_limit_handler(error):
```

Ruff (0.13.1)

Unused function argument: `error`

(ARG001)